### PR TITLE
Make SDK build of OmniSharp the default

### PR DESCRIPTION
**Pull request by JoeRobich**
In preparation for taking a new build of O# without the included Mono/MSBuild.

- Default `useModernNet` option to `true`
- Remove `useGlobalMono` option
- Simplify O# launching
- Move to new VS Code testing package

Part of https://github.com/OmniSharp/omnisharp-vscode/issues/5120

### CODE REVIEW

**Comment by filipw** [comment]
very exciting 😅

File: package.json [hunk 1]
```diff
@@ -1,4051 +1,4042 @@
...
-    },
-    "viewsWelcome": [
-      {
-        "view": "debug",
-        "contents": "[Generate C# Assets for Build and Debug](command:dotnet.generateAssets)\n\nTo learn more about launch.json, see [Configuring launch.json for C# debugging](https://aka.ms/VSCode-CS-LaunchJson).",
-        "when": "debugStartLanguage == csharp"
-      }
-    ]
-  }
+	"name": "csharp",
```
Comment (333fred) [inline package.json:2]: Why is there an 8000 line diff in this file?

Comment (JoeRobich) [reply package.json:2]: oh no. indentation went from 2 to 4. sigh

File: package.json [hunk 2]
```diff
@@ -687,17 +687,17 @@
                   "items": {
                     "type": "string"
                   },
-                  "description": "Array of symbol server URLs (example: http\u200b://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
```
[the remark this answers is not in the document]
Comment (JoeRobich) [reply package.json]: Reverted

File: package.json (hunk 1 above)
Comment (JoeRobich) [reply package.json]: Fixed

**Review by 333fred — APPROVED** [review]

**Pull request merged**